test/mpi/perf/Makefile.am

```diff
@@ -22,5 +22,3 @@ dtpack_CFLAGS    = -O
 nestvec_CFLAGS   = -O
 nestvec2_CFLAGS  = -O
 indexperf_CFLAGS = -O
-
-twovec_LDADD = -lm $(top_builddir)/util/mtest.o
```

Comment:
The commit message for https://github.com/pmodels/mpich/pull/4429/commits/e8c26f8ec9c1a204a468297fac3284b3cab30418 need be updated.

Reply:
Thanks. I will change it.